src/sth_helper.js

```diff
@@ -140,15 +140,43 @@
     return sthConfig.OPERATION_TYPE_PREFIX + request.method.toUpperCase();
   }
 
-  module.exports = function (aSthConfig) {
+  /**
```

Comment:
I agree on adding this helping function here... but was it removed from its original location? I think it was already added somewhere before... Correct me if I'm wrong (most probably :))